.agent/skills/ss14-ecs-components/SKILL.md

```
@@ -0,0 +1,346 @@
+>
+> [DataField("baseSpeed")]
+> public float BaseSpeed;
+>
+> // ✅ Правильно — имя выводится автоматически
+> [DataField]
+> public int Counter;        // → counter в YAML
+>
+> [DataField]
+> public float BaseSpeed;    // → baseSpeed в YAML
+> ```
+
```

markdownlint-cli2 (0.20.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)